src/annotator/highlighter.js

```diff
@@ -264,6 +274,57 @@ export function removeHighlights(highlights) {
   }
 }
 
+/**
+ * Set whether the given highlight elements should appear "focused".
+ *
+ * A highlight can be displayed in a different ("focused") style to indicate
+ * that it is current in some other context - for example the user has selected
+ * the corresponding annotation in the sidebar.
+ *
+ * @param {HighlightElement[]} highlights
+ * @param {boolean} focused
+ */
+export function setHighlightsFocused(highlights, focused) {
+  highlights.forEach(h =>
+    h.classList.toggle('hypothesis-highlight-focused', focused)
```

Comment:
Yes, although we have to be a bit careful about unwanted effects of page CSS here, eg. in case the page's CSS were to define its own `.is-focused` selector. We could perhaps have a convention like prefixing the `is-<state>` class with `hypothesis-` as well.